google_bigquery/data/google_bigquery_location_data.xml

```diff
@@ -0,0 +1,122 @@
+    </record>
+    <record id="location_13" model="google.bigquery.location">
+        <field name="name">asia-east2</field>
+        <field name="description">Hong Kong</field>
+    </record>
+    <record id="location_14" model="google.bigquery.location">
+        <field name="name">asia-southeast2</field>
+        <field name="description">Jakarta</field>
+    </record>
+    <record id="location_15" model="google.bigquery.location">
+        <field name="name">australia-southeast2</field>
+        <field name="description">Melbourne</field>
+    </record>
+    <record id="location_16" model="google.bigquery.location">
+        <field name="name">asia-south1</field>
+        <field name="description">Mumbai</field>
+    </record>
+    <record id="location_17" model="google.bigquery.location">
+        <field name="name">asia-northeast2</field>
+        <field name="description">Osaka</field>
+    </record>
+    <record id="location_18" model="google.bigquery.location">
+        <field name="name">asia-northeast3</field>
+        <field name="description">Seoul</field>
+    </record>
+    <record id="location_19" model="google.bigquery.location">
+        <field name="name">asia-southeast1</field>
+        <field name="description">Singapore</field>
+    </record>
+    <record id="location_20" model="google.bigquery.location">
+        <field name="name">australia-southeast1</field>
+        <field name="description">Sydney</field>
+    </record>
+    <record id="location_21" model="google.bigquery.location">
+        <field name="name">asia-east1</field>
+        <field name="description">Taiwan</field>
+    </record>
+    <record id="location_22" model="google.bigquery.location">
+        <field name="name">asia-northeast1</field>
+        <field name="description">Tokyo</field>
+    </record>
+
+    <record id="location_23" model="google.bigquery.location">
+        <field name="name">europe-west1</field>
+        <field name="description">Belgium</field>
+    </record>
+    <record id="location_24" model="google.bigquery.location">
+        <field name="name">europe-north1</field>
+        <field name="description">Finland</field>
+    </record>
+    <record id="location_25" model="google.bigquery.location">
+        <field name="name">europe-west3</field>
+        <field name="description">Frankfurt</field>
+    </record>
+    <record id="location_26" model="google.bigquery.location">
+        <field name="name">europe-west2</field>
+        <field name="description">London</field>
+    </record>
+    <record id="location_27" model="google.bigquery.location">
+        <field name="name">europe-west4</field>
+        <field name="description">Netherlands</field>
+    </record>
+    <record id="location_28" model="google.bigquery.location">
+        <field name="name">europe-central2</field>
+        <field name="description">Warsaw</field>
+    </record>
+    <record id="location_29" model="google.bigquery.location">
```

Comment:
Maybe it makes sense to put the description `europe_west6` in the name eg `location_europe_west6`